common/src/main/java/muramasa/antimatter/data/AntimatterMaterialTypes.java

```diff
@@ -47,6 +47,10 @@ public class AntimatterMaterialTypes {
     public static MaterialTypeItem<?> GEM = new MaterialTypeItem<>("gem", 2, true, Ref.U);
     public static MaterialTypeItem<?> GEM_BRITTLE = new MaterialTypeItem<>("gem_brittle", 2, true, Ref.U);
     public static MaterialTypeItem<?> GEM_POLISHED = new MaterialTypeItem<>("gem_polished", 2, true, Ref.U);
+    public static MaterialTypeItem<?> GEM_EXQUISITE = new MaterialTypeItem<>("gem_polished", 2, true, Ref.U);
+    public static MaterialTypeItem<?> GEM_FLAWLESS = new MaterialTypeItem<>("gem_polished", 2, true, Ref.U);
+    public static MaterialTypeItem<?> GEM_FLAWED = new MaterialTypeItem<>("gem_polished", 2, true, Ref.U);
+    public static MaterialTypeItem<?> GEM_CHIPPED = new MaterialTypeItem<>("gem_polished", 2, true, Ref.U);
```

Comment:
I think you forgot to change the ids of these new material types

Reply:
bruh